.github/workflows/daily_report.yml

```
@@ -0,0 +1,73 @@
+name: Daily Report Schedules
```

The file name should change to `agora_daily_report.yml` 

```suggestion
name: Agora Daily Report Schedules
```